src/views/Dissolution/DissolutionAffidavit.vue

```diff
@@ -22,17 +22,6 @@ export default class DissolutionAffidavit extends Mixins(CommonMixin) {
   @Getter getAffidavitStep!: UploadAffidavitIF
   @Getter getShowErrors!: boolean
 
-  @Action setIgnoreChanges!: ActionBindingIF
-
-  /** Called when component is created. */
-  private created (): void {
-    // ignore data changes until page has loaded
-    this.setIgnoreChanges(true)
-    Vue.nextTick(() => {
-      this.setIgnoreChanges(false)
-    })
-  }
```

Comment:
I believe this is no longer needed since the data is now loaded before the page is created.